app/views/branding_groups/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for(:html_title) { 'Collections - Single Cell Portal' } %>
```

Comment:
I can see html_title for Upload but not for Collections... Also not seeing identifying titles for the individual collection pages..
![Screen Shot 2022-08-18 at 10 29 49 AM](https://user-images.githubusercontent.com/6005195/185421006-0883702b-4988-4217-9038-c12051d06d66.png)
.

Reply:
Oh nice catch - clicking to the Features and the Collections pages from the dropdown profile menu vs clicking the buttons from the home page bring you to different pages (which I hadn't noticed) so I've updated those. Also added titles for the collections.

<img width="1473" alt="Screen Shot 2022-08-18 at 11 16 01 AM" src="https://user-images.githubusercontent.com/54322292/185432966-ba338b52-39f1-45df-993b-8f597b3d0b41.png">

Reply:
<img width="582" alt="Screen Shot 2022-08-18 at 11 25 22 AM" src="https://user-images.githubusercontent.com/54322292/185433443-14a7a2ff-6de7-4040-b950-6342f92de711.png">

I wonder if the naming convention for Collections should be "<collection title> - Collections - Single Cell Portal" or "<collection title> - Single Cell Portal" though?

Reply:
How about "Collection title - Single Cell Portal Collections"

Reply:
Would be a nice time to have a UX partner to check in on this haha, from what I could find online the general convention is to keep the web app name ("Single Cell Portal") and the end as the overarching title, and then chain subsequent section titles in order of most granular/specific title being the leftmost. So I think following that we would want "Collection title - Collections - Single Cell Portal"

Looking at this in the history i do think this looks okay as it basically adds an extra descriptor each level you drill down.

<img width="427" alt="Screen Shot 2022-08-18 at 1 37 44 PM" src="https://user-images.githubusercontent.com/54322292/185459870-02bdb978-5016-4ac8-9986-a809e74d6647.png">